src/Common/src/Interop/Linux/cgroups/Interop.cgroups.cs

```diff
@@ -113,7 +154,7 @@ private static string FindCGroupPath(string subsystem)
         /// <param name="root">The path of the directory in the filesystem which forms the root of this mount; null if not found.</param>
         /// <param name="path">The path of the mount point relative to the process's root directory; null if not found.</param>
         /// <returns>true if the mount was found; otherwise, null.</returns>
-        private static bool TryFindHierarchyMount(string subsystem, out string root, out string path)
+        private static bool TryFindHierarchyMount(string subsystem, out CGroupVersion version, out string root, out string path)
```

Comment:
We need to know the version of cgroup because that affects the information we are looking for when parsing `/proc/self/cgroup` later on.